### PR TITLE
Fix WAL Exception catch & support new DataTypes in QueryDataSetUtils

### PR DESCRIPTION
**Pull request by jt2594838**
One Tablet deserialization does not support new types in QueryDataSetUtils, which causes WAL recovery failure.
TsDataType switches are supplemented with all cases to indicate they are CONSIDERED rather than FORGOTTEN.

### CODE REVIEW

File: integration-test/src/main/java/org/apache/iotdb/db/query/udf/example/UDAFAvg.java [hunk 1]
```diff
@@ -101,6 +101,8 @@ public void addInput(State state, Column[] columns, BitMap bitMap) {
         addDoubleInput(avgState, columns, bitMap);
         return;
       case TEXT:
+      case STRING:
+      case BLOB:
```
Comment (shuwenwei) [inline integration-test/src/main/java/org/apache/iotdb/db/query/udf/example/UDAFAvg.java:105]: Missing data type TIMESTAMP and DATE

File: integration-test/src/main/java/org/apache/iotdb/db/query/udf/example/UDAFSum.java [hunk 2]
```diff
@@ -101,6 +101,8 @@ public void addInput(State state, Column[] columns, BitMap bitMap) {
         addDoubleInput(sumState, columns, bitMap);
         return;
       case TEXT:
+      case STRING:
+      case BLOB:
```
Comment (shuwenwei) [inline integration-test/src/main/java/org/apache/iotdb/db/query/udf/example/UDAFSum.java:105]: Missing data type TIMESTAMP and DATE

File: iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/udf/builtin/UDTFAbs.java [hunk 3]
```diff
@@ -89,6 +97,14 @@ public Object transform(Row row) throws IOException {
         return Math.abs(row.getFloat(0));
       case DOUBLE:
         return Math.abs(row.getDouble(0));
+      case VECTOR:
+      case UNKNOWN:
```
Comment (shuwenwei) [inline iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/udf/builtin/UDTFAbs.java:101]: Should we add VECTOR and UNKNOWN types to other switch case statements?

File: iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties [hunk 4]
```diff
@@ -27,21 +27,21 @@ cluster_name=defaultCluster
 ### Seed ConfigNode
 ####################
 
-cn_seed_config_node=127.0.0.1:10710
+cn_seed_config_node=nel-22:10710
```
Comment (shuwenwei) [inline iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties:30]: It also needs to be modified in iotdb-system.properties.template

**Review by shuwenwei — APPROVED** [review]

File: integration-test/src/main/java/org/apache/iotdb/db/query/udf/example/UDAFSum.java (hunk 2 above)
Comment (jt2594838) [reply integration-test/src/main/java/org/apache/iotdb/db/query/udf/example/UDAFSum.java:105]: Added.

File: iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/udf/builtin/UDTFAbs.java (hunk 3 above)
Comment (jt2594838) [reply iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/udf/builtin/UDTFAbs.java:101]: Removed.

File: iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties (hunk 4 above)
Comment (jt2594838) [reply iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties:30]: Reverted.

**Review by shuwenwei — APPROVED** [review]

**Pull request merged**